sklearn/gaussian_process/tests/test_gpc.py

```diff
@@ -62,15 +63,16 @@ def test_lml_precomputed():
 def test_converged_to_local_maximum():
     """ Test that we are in local maximum after hyperparameter-optimization."""
     for kernel in kernels:
-        if kernel == fixed_kernel: continue
+        if kernel == fixed_kernel:
+            continue
         gpc = GaussianProcessClassifier(kernel=kernel).fit(X, y)
 
         lml, lml_gradient = \
             gpc.log_marginal_likelihood(gpc.kernel_.theta, True)
 
-        assert_true(np.all((np.abs(lml_gradient) < 1e-4)
-                           | (gpc.kernel_.theta == gpc.kernel_.bounds[:, 0])
-                           | (gpc.kernel_.theta == gpc.kernel_.bounds[:, 1])))
```

Comment:
Just a note for later: let's ignore those kind of non-important pep8 violations. We could give a list of flake8 warnings to ignore in a conf file in the repo.